ios-base/DroidKaigi 2020/Announcements/Views/Cells/AnnouncementCell.swift

```diff
@@ -0,0 +1,26 @@
+import ios_combined
+import UIKit
+
+final class AnnouncementCell: UITableViewCell {
+    enum Constant {
+        static let dateFormatter: DateFormatter = {
+            let formatter = DateFormatter()
+            formatter.locale = Locale.current
+            formatter.calendar = Calendar.current
+            formatter.dateFormat = DateFormatter.dateFormat(fromTemplate: "dMMM HH:mm", options: 0, locale: Locale.current)
+            return formatter
+        }()
+    }
+
+    @IBOutlet weak var iconImageView: UIImageView!
+    @IBOutlet weak var publishedAtLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var contentLabel: UILabel!
+
+    func configura(_ announcement: Announcement) {
+        // TODO: Set icon image
```

Comment:
@takahirom 

I must set icon image for announcement type. Where can I icon image? I don't find in Figma file.

Reply:
Probably that from 2019 👀 But when I talked to the designer, it was a good idea to use the material design icons.
https://github.com/DroidKaigi/conference-app-2019

Reply:
Thanks!
I'll use material design icons.

- [warning_amber_24px](https://www.figma.com/file/boeZxc72hgSA3Zz9jFOBt5/material-design-icons-outlined?node-id=0%3A3029)
- [error_outline_24px](https://www.figma.com/file/boeZxc72hgSA3Zz9jFOBt5/material-design-icons-outlined?node-id=0%3A3003)
- [assignment_24px](https://www.figma.com/file/boeZxc72hgSA3Zz9jFOBt5/material-design-icons-outlined?node-id=0%3A307)